### PR TITLE
Update functionality for runtime

### PR DESCRIPTION
**Pull request by matthew-zhao**
Allows users to update runtime through UI

### CODE REVIEW

File: runtime/runtime.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from grizzly import *
 import usb
 import os
+from base64 import b64decode
```
Comment (AllanYangZhou) [inline runtime/runtime.py]: We don't need this anymore right?

**Comment by nikitakit** [comment]
Can you guys merge this into my branch, and not the other way around?

**Comment by nikitakit** [comment]
Or better yet, cherry-pick some of this stuff on top of my develop branch.

**Comment by matthew-zhao** [comment]
Alright so how do I do that in a nice way? I've never cherry picked before

**Comment by nikitakit** [comment]
What you can do is check out my branch, and then cherry-pick your new commits one-by-one. Probably start with `git cherry-pick 49357b8` and then do `git cherry-pick aea2ff8`.

Once you're done, you can push the results to a branch on github

**Comment by matthew-zhao** [comment]
Alright, so I've done that (branch 'nikita' on my fork). Should I wait for your PR to be merged in first before making a PR for my branch?

**Comment by nikitakit** [comment]
This code is now on my branch.

**Pull request closed without merging**